src/lib/Rattletrap/Utility/Json.hs

```diff
@@ -33,7 +33,7 @@ optional ::
   Aeson.Parser (Maybe value)
 optional object key = object Aeson..:? Key.fromString key
 
-pair :: (Aeson.ToJSON value, Aeson.KeyValue pair) => String -> value -> pair
+pair :: (Aeson.ToJSON value, Aeson.KeyValue e pair) => String -> value -> pair
```

Comment:
This will _only_ work for aeson 2.2. In order to continue supporting older versions, I'll either need CPP or some other solution. Obviously I could use `(.=)` directly, but the point here was to provide a named version of that operator.